lib/pages/settings_pages/settings_page.dart

```diff
@@ -298,12 +296,25 @@ class _SettingsPageState extends State<SettingsPage> {
     );
   }
 
+  languagesSortedAlphabetically() {
```

Comment:
I'd add the return type in front of the function and two arguments, not to depend on variables defined in the file itself
`List<Language> languagesSortedAlphabetically(BuildContext context, List<Language> languageCodes)`